src/commcare_cloud/commands/ansible/run_module.py

```diff
@@ -296,3 +296,40 @@ def run(self, args, unknown_args):
             become_user='cchq',
             extra_args=['-e', f'@{group_vars}'] + unknown_args,
         )
+
+
+class ListReleases(CommandBase):
+    command = 'list-releases'
+    help = 'List names that can be passed to `deploy --resume=RELEASE_NAME`'
```

Comment:
would be nice if this filtered out ones that were 'complete' (or perhaps just included that info in the list).

Reply:
:+1: I'll implement that in a follow-up PR.

FWIW, any "incomplete" build should build to completion on `--resume`, and if you're watching the terminal output you should have time to cancel if a particular resume starts doing some unexpected build operations. Any step that "changes" prior to "Check release" would indicate the build had not previously completed. But granted, this requires intimate knowledge of the build, so not a reasonable expectation for everyone.

Reply:
https://github.com/dimagi/commcare-cloud/pull/5854